includes/options/class-amp-options-manager.php

```diff
@@ -199,12 +200,22 @@ public static function is_website_experience_enabled() {
 	 *
 	 * @since 1.2
 	 *
+	 * @param bool $include_posts_check Include check to see if any `amp_story` posts exists.
 	 * @return bool Enabled.
 	 */
-	public static function is_stories_experience_enabled() {
+	public static function is_stories_experience_enabled( $include_posts_check = true ) {
+		$has_stories_posts = true;
+
+		// Note: A post count cannot be done until the post type is registered (see AMP_Story_Post_Type::register).
+		if ( $include_posts_check ) {
+			$story_posts_count       = (array) wp_count_posts( AMP_Story_Post_Type::POST_TYPE_SLUG );
```

Comment:
This doesn't seem quite right. I have a draft Story post and it is is not getting picked up here.

Reply:
Actually, this is not working for me even for published `amp_story` posts.

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/4219/commits/e1aece941277e5d3a2f2c3ced1950c56b6e6609a.